### ISSUE TITLE
Morphed Backend Relations

### ISSUE THREAD

**Issue by ab192130** [octobercms/october#1359]
Hi OctoberCMS Team,

I have a Eloquent Model with morphed relations. **Product** model morphs to the models like **Car** and **Estate**. The table for products looks like this:

```
| id | title         | object_id | object_type |
|----|---------------|-----------|-------------|
| 1  | Mercedes Benz | 1         | car         |
```

How can I manage product model in my backend with **Backend Relations** feature? I tried it but it throws an error like "unknown variable - $widget". 

What is your best practices with it? Thanks...

**Comment by ARH-Digital** [comment]
Unfortunately they do not permit morphic relations in the backend.
https://octobercms.com/docs/backend/relations#relationship-types

This can be fixed by adding morphs into the RelationController.php
I will look at fixing this and submitting a pull request. Hopefully they accept it as I have used this a couple of times now.

**Comment by ab192130** [comment]
Thanks you, @ARH-Digital. It made up my day. I will look forward to it.

**Comment by daftspunk** [comment]
Fixed by #1741

**Comment by Arekusei** [comment]
Many To Many Polymorphic relations are not supported yet.

In file https://github.com/octobercms/october/blob/develop/modules/backend/behaviors/RelationController.php#L1341 presents duplicated code for morph relations.

**Comment by daftspunk** [comment]
Thanks @Arekusei, please submit a Pull Request to the [test plugin for investigation](https://github.com/daftspunk/oc-test-plugin) that demonstrates the issue. Don't forget to include step by step instructions how how to replicate it *using the test plugin*.

**Comment by Arekusei** [comment]
@daftspunk Okay, I'll do later pull request for the test module.

**Comment by Arekusei** [comment]
@daftspunk, done.

**Comment by LukeTowers** [comment]
@Arekusei Should be fixed by #2275: see https://github.com/octobercms/october/issues/2561#issuecomment-277541963 for further information
